0004-post-tags.md

```diff
@@ -0,0 +1,335 @@
+
+# Prior art
+
+Similar implementations can be found on Reddit as well as a great many image hosting platforms.
+
+Reddit's Flair System is rather restrictive, a Post can only have a single custom Flair (such as "News") greatly limiting the usefullness of the feature.
+
+Pros:
+- low moderation effort due to predefined tags
+- no/little content fragmentation since typos are not possible
+- no flair spam due to only 1 flair being allowed at a time
+
+Cons:
+- extreme moderation effort for 1st time setup due to inability to mix flairs
+- filter lists get extremely long because all variants of a flair need to be filtered (for example "News Europe", "News America", etc. instead of just "News")
+- flairs need to be created before being usable
+
+Tag/Flair Systems on Image hosting platform generally tend to favor a more unmoderated experience, often allowing Users to type in the tags themselves.
+
+Pros:
+- enhanced user freedom when applying tags
+- no delay between arising demand for tag and creation of the tag
+
+Cons:
+- greater moderation demand since tags have to be checked for inappropriate content
+- very high chance for content fragmentation (typos, differences in naming things, language barriers)
+- almost completely removes the ability of users to blocklist content tags (pronography would slip past a "pornography" blocklist).
+
+# Unresolved questions
+
+- How would tags federate/display on other Fediverse services such as Mastodon?
+- ~~Should instance wide and community tags be implemented separately or together. If separately which one should be first.~~
+-> resolved, community tags will be implemented first with instance tags to follow some time later.
+- There is a discussion about creation permissions for tags however that changes nothing about the fundamental functionality and can be discussed after initial experience is gathered.
+- Should there be a limit on how many tags can be applied? If so, should this limit be hard coded or community configurable?
+-> should be made configurable, still up for debate whether to be implemented with this RFC or later on.
+
+# Future possibilities
+### Global Tags
```

Comment:
Goes from `#` to `###`, it should go to `##`

Reply:
Should be done with 6257bc3e86bc40a00f26f2a4dd56d15f44209fa7. To be perfectly honest I have no idea why on earth I skipped the headings back when I first wrote this (it seems far too consistent to be random), I noticed it when I was merging the other changes but didn't bother fixing it then. Weird decision from past me....